R/comparison.R

```diff
@@ -0,0 +1,93 @@
+rcmdcheck_comparison <- function(old, new) {
+  stopifnot(is.list(old))
+  stopifnot(inherits(new, "rcmdcheck"))
+
+  # Generate data frame of problems
+  old_df <- do.call(rbind, lapply(old, as.data.frame, which = "old"))
```

Comment:
It would be good to keep the names of `old` somewhere, because for CRAN checks, that's the CRAN platform name. `cran_check_results` should put it in returned objects, ideally.

We can do this after merging.